src/server/pfs/server/driver.go

```diff
@@ -113,26 +113,23 @@ const (
 	tombstone = "delete"
 )
 
-// Instead of making the user specify the respective size for each cache,
-// we decide internally how to split cache space among different caches.
-//
-// Each value specifies a percentage of the total cache space to be used.
 const (
```

Comment:
I'm not sure how much a comment here is worth, it's not particularly discoverable if it's just on the default value we use, I think it might make more sense on the `NewAPIServer` function.